src/components/addTodoForm/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import { addNewTodo, isAdding } from "redux/actions"
+import ImageButton from "components/buttons/imageButton"
+import "components/addTodoForm/form.scss"
+import { customSanitize } from "utils/sanitizeInput"
+import { DELETE_BUTTON_ALT, IMAGE_PATHS } from "utils/constants"
+
+function AddTodoForm() {
+  const dispatch = useDispatch()
+  const [taskTitle, setTaskTile] = useState("")
+  const [isInputFieldEmpty, setIsInputFieldEmpty] = useState(false)
+
+  function handleAddTodo(event) {
+    event.preventDefault()
+    if (taskTitle.trim() === "") {
+      setTaskTile("")
+      setIsInputFieldEmpty(true)
+    } else {
+      const satitizedText = customSanitize(taskTitle.trim())
+      if (satitizedText !== "") {
+        dispatch(addNewTodo(customSanitize(taskTitle.trim())))
+        setTaskTile("")
+        dispatch(isAdding(false))
+      } else {
+        setTaskTile("")
+        setIsInputFieldEmpty(true)
+      }
+    }
+  }
+
+  return (
+    <form className="todo-form__container" onSubmit={handleAddTodo}>
+      <textarea
+        className="todo-form__container__textarea"
+        name="body"
+        placeholder="Add Todo"
+        value={taskTitle}
+        onChange={(e) => setTaskTile(e.target.value)}
+        required
```

Comment:
```suggestion
```

Let's remove this as we already have empty task checker implemented. Thanks.

Reply:
Bhaia if i remove required it will no longer show the default warning message

https://github.com/umh-wtag/todo-app/assets/139974355/424048c7-33d5-43e4-954c-69cfa3211b2c

Reply:
Yes, but we do have a empty task error message to display on the right side of our delete button, which do the job for us. Right?